manual.typ

```diff
@@ -230,23 +230,25 @@ Draws a rectangle to the canvas.
   ```])
 
 === Arc
-Draws an arc to the canvas.
+Draws an arc to the canvas. Exactly two of the three values `start`, `stop`, and `delta` should be defined.
 ```typ
-#arc(position, start, stop, radius: 1, name: none, anchor: none, fill: auto, stroke: auto)
+#arc(position, start: auto, stop: auto, delta: auto, radius: 1, name: none, anchor: none, fill: auto, stroke: auto, mode: "OPEN")
 ```
 #def-arg("position", `<coordinate>`, [The coordinate to start drawing the arc from.])
 #def-arg("start", `<angle>`, [The angle to start the arc.])
 #def-arg("stop", `<angle>`, [The angle to stop the arc.])
-#def-arg("radius", `<number>`, [The angle to stop the arc.])
+#def-arg("delta", `<angle>`, [The angle that is added to start or removed from stop.])
+#def-arg("radius", `<number>`, [The radius of the arc.])
+#def-arg("mode", `<string>`, [The options are "OPEN" (the default, just the arc), "CLOSE" (a circular segment) and "PIE" (a circular sector). The filling will depend whether the local `fill` variable is set to other that `none`, or if set to `auto`, the global context variable `fill` is set to other than `none`.])
 
 #example({
```

Comment:
As a nice to have could you add extra examples? e.g. an arc with "CLOSE" using start and delta, and an arc with "PIE" using stop and delta.

Don't worry if you don't want to :)

Reply:
No problem. Anyway, I have to make a little small change in the code, so PIE is closed correctly.